test-files/test.org

```diff
@@ -0,0 +1,90 @@
+* Simple note
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:END:
+
+** Front
+Simple note body
+
+** Back
+Lorem
+
+* Formatted note (formatted)
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:ANKI_FORMAT: t
+:END:
+
+** Front
+/Simple/ *note* =body=
+
+** Back
+
+#+begin_example
+Lorem ipsum
+dolor sit
+amet
+#+end_example
+
+* Formatted note (plain)
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:ANKI_FORMAT: nil
+:END:
+
+** Front
+/Simple/ *note* =body=
+
+** Back
+
+#+begin_example
+Lorem ipsum
+dolor sit
+amet
+#+end_example
+
+
+* Note without Back
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:END:
+
+** Front
+Simple note body
+
+* Note with attachment
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:DIR: test-attachments
+:END:
+
+[[attachment:1x1.gif][Test image]]
+
+* Note with file link
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Basic
+:ANKI_DECK: Tests
+:END:
+
+** Front
+Note with file link
+
+** Back
+[[file:test-attachments/1x1.gif][Test image]]
+
+
+* Cloze with text
+:PROPERTIES:
+:ANKI_NOTE_TYPE: Cloze
+:ANKI_DECK: Tests
+:END:
+** Text
+Lorem {{c1::lorem}}.
+
+* Whatever
+If I remove this - my anki export freezes
```

Comment:
This should hopefully be fixed by https://github.com/anki-editor/anki-editor/commit/c2d66f81382d0c0eb948f67c2cb8152e2b800f59